src/component/base/DraftEditor.react.js

```diff
@@ -190,6 +191,10 @@ class DraftEditor extends React.Component {
       if (!this.props.readOnly) {
         const method = this._handler && this._handler[eventName];
         method && method(this, e);
+      } else if (eventName === 'onCopy') {
+        editOnSelect(this)
```

Comment:
When draftjs doc is in readonly state, it's selection is always 0 (from what I've observed). This forces it to set the selection, which then allows copy to do the right thing. If we don't do this and still let copy go through draft, nothing will be copied.